corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -383,6 +383,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             'click @ui.paginators': 'paginateAction',
             'click @ui.paginationGoButton': 'paginationGoAction',
             'click @ui.columnHeader': 'columnSortAction',
+            'keypress @ui.columnHeader': 'columnSortAction',
```

Comment:
If this is for case lists with tiles only, should this not go [here](https://github.com/dimagi/commcare-hq/blob/519262abf6b506c62ee9d8fc131aac42fd36b46b/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js#L808)?

Reply:
The new "sort by" dropdown is only for case tile but this adds accessibility so the "enter" key press will trigger the same event as the "click" so it is also applicable to case list. So I think this should stay here. Open to talk about it more if you disagree